src/interfaces/store-interfaces/state-interfaces/business-information-interface.ts

```diff
@@ -9,4 +9,5 @@ export interface BusinessInformationIF {
   naicsCode?: string
   naicsDescription?: string
   naicsKey?: string // from API
+  incorporationAgreementType?: string // BENs only
```

Comment:
I'm assuming this will be returned in the business object. This will be needed to identify incorporation agreement type changes in BEN corrections.

^^ There's a BE ticket to add this.

Reply:
I took a brief look at this last week and we can grab this info from the IA filing json.  It doesn't look like we save it to a specific db field in LEAR.

Reply:
actually, i don't know enough about incorporation agreement type.  we'll need to talk about it more when we groom that ticket.

Reply:
Yes, this property is in the original IA, but near-future corrections won't load the original filing to be corrected -- they will fetch a current business snapshot, so the type of agreement used to incorporate the company needs to be available some other way, like this maybe.